vendor/composer/autoload_static.php

```diff
@@ -61,6 +61,10 @@ class ComposerStaticInit22ddb69348c7ed922c96325249cef3d0
         array (
             'Psr\\Log\\' => 8,
```

Comment:
I have phing installed globally, that shouldn't matter in this case. I'm uploading a PR now which only deals with composer stuff